### PR TITLE
fix: support event types

### PR DESCRIPTION
**Pull request by daybrush**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -317,3 +317,58 @@ export type ExcludeKeys = keyof Component
...
+	replace: (src: string) => void;
+	replaceItem: (content: string) => void;
+	remove: () => void;
+	removeItem: () => void;
+};
+export type OnRender = {
+	next: () => void;
+};
+
+export type OnChnage = {
```
Comment (WoodNeck) [inline src/types.ts:361]: Typo here

**Review by WoodNeck — APPROVED** [review]
LGTM
We should update types for the @egjs/lazyloaded I guess?

File: src/RenderManager.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export default class RenderManager {
 		}).on("error", ({
 			target,
 			itemIndex,
-		}) => {
+		}: any) => {
```
Comment (mixed) [inline src/RenderManager.ts:52]: Where is `@egjs/lazyloaded`?

**Comment by WoodNeck** [comment]
As @egjs/component is dependent on typescript^3.0.0, this module should be updated also.

**Review by mixed — APPROVED** [review]
I think that `lazyloaded` should be update.

**Comment by daybrush** [comment]
@mixed 
After merging, I will add a function related to lazyloaded(#354) and then post the PR again.

**Pull request merged**